helm/prometheus/locals.tf

```diff
@@ -0,0 +1,19 @@
+locals {
+  helm_stable_repository = "https://kubernetes-charts.storage.googleapis.com"
+
+  storage_class_defaults = {
+    "aws" = "gp2"
+    "gcp" = "standard"
+  }
+  storage_class = merge(local.storage_class_defaults, var.storage_class)
+
+  prometheus_helm_defaults = {
+    "server.persistentVolume.storageClass"       = lookup(local.storage_class, var.cloud_provider, "gp2")
+    "server.persistentVolume.size"               = "500Gi" # Without any idea of what normal is I'm just setting this as a random value
+    "server.retention"                           = "7d"
```

Comment:
hmm we should think a bit about these defaults; if we're going to have a default PV size of 500GB we should have a much larger default retention period, like maybe 6mos. But the way I like to think of default values is: "how would I want my dev cluster configured?" -- from that perspective, I like the idea of having small default values in the module then override them with the larger values in the implementation. That way no one accidentally creates unnecessarily expensive resources. I think that makes sense here too because every cluster will have a prometheus, but all by one of the clusters should be using either small or default values, since they'll be federating to admin-1, which is the only cluster that should have larger PVs and retention value.

Reply:
This module is just a generic "get prometheus up and running on your cluster" module I think instead what we should be doing is dropping down the value to a more sane number for the prometheus instance running on every cluster.

For the `admin-1` cluster aka the federator we should set the `var.prometheus_settings` map to the appropriate value we want

Reply:
agreed completely.

Reply:
Fixed in e74fd33a9769dc7e41f381720e2a225bdc05c9d1